javascript.md

```diff
@@ -686,52 +690,3 @@ self.$el.find('.my-thing').on('click', function() {
   
```

Comment:
Leaving for now — it's in the "needs review" section and we can address as we continue to clean up.